.github/workflows/deploy.yml

```diff
@@ -9,10 +9,15 @@ jobs:
 
     steps:
       - uses: actions/checkout@master
+
+      - name: Embed version
+        run: |
+          echo -e "\ncachedversion = \"Workshop ${GITHUB_SHA:0:7}\"" >> lua/wire/server/wirelib.lua
```

Comment:
This will append to the end of wirelib, but wirelib’s final line before appending is a return so wont this result in a syntax error?